crates/ruma-events/src/room/message/audio.rs

```diff
@@ -18,7 +18,7 @@ pub struct AudioMessageEventContent {
     /// uploaded file. Otherwise, this should be interpreted as a user-written media caption.
     pub body: String,
 
-    /// Formatted form of the message `body`, if `body` is a caption.
+    /// Formatted form of the message `body`.
```

Comment:
Why did you remove that?

Reply:
Because it just adds confusion IMO. This implies that the field is something else if the body is not a caption (formatted filename? 😅), but AFAIU you're simply not supposed to use it if the body is not a caption. Maybe that could be said here.

Reply:
Yes, I believe we should keep some explanation for when the field can be used.

Reply:
Do you think it should be in the tagline as before, or is a separate paragraph okay? Also, any suggested wording?

Reply:
It's fine in a separate paragraph. Maybe something like:

> This should only be set if the body represents a caption.